services/swap/passThroughTokenSwap.ts

```diff
@@ -13,30 +13,33 @@ type PassThroughTokenSwapArgs = {
   price: number
   slippage: number
   senderAddress: string
-  swapAddress: string
-  outputSwapAddress: string
+  inputPool: PoolEntityType
+  outputPool: PoolEntityType
   tokenA: TokenInfo
   client: SigningCosmWasmClient
 }
 
 export const passThroughTokenSwap = async ({
   tokenAmount,
   tokenA,
-  outputSwapAddress,
-  swapAddress,
+  outputPool: outputSwap,
+  inputPool: swap,
```

Comment:
Sort of minuscule- why renaming it to different variables if can just use whatever you pass in here?